### PR TITLE
Reworking DPC issuance description

### LINKED ISSUE #237: DoD issuer of PIV card and of derived credential definition too narrow
### All Fields Are Required
**Organization Name (N/A, if individual)**: DoD

**Organization Type (see below for codes)**: 1 - Federal
 
**Reference (Include section/paragraph or pdf line number)**: 2.10.1 Line 1111
 
**Comment (Include rationale for comment)**: This Section requires the issuer of the PIV card and of a derived credential be one and the same entity; this definition is too narrow to account for Agencies where the issuers of derived credentials are not the organization that manages the PIV issuance.  (REF: The "[d]erived PIV credentials SHALL be bound to the cardholder's PIV account only by the organization that manages that PIV account", and the binding is described as follows, "[i]issuance of a derived PIV credential is an instance of the post-enrollment binding".)
 
**Suggested Change**: "DoD recommends changing 1111-1113 as follows, ""Derived PIV credentials SHALL be bound to the cardholder’s PIV account only by the organization
that manages that PIV account life-cycle management bound to the cardholder’s PIV account or eligibility."""
 
---
 
Organization Type: 1 = Federal, 2 = Industry, 3 = Academia, 4 = Self, 5 = Other

### PR DESCRIPTION
**Pull request by regenscheid**
**This is a work in progress.**

Intended to close #237.

I believe the heart of this issue is to describe how shared services (e.g., USAccess, who handles card issuance for many agencies) fit into Derived PIV issuance.  In particular, many readers appear to forget that the term "issuer" applies to the department/agency responsible for issuance, not the entity that may be actually handing over the card.  

I don't think this PR is ready yet.  For one, it's awkward.  But more significantly, it's using terms like "the issuing department or agency" and "shared services" that aren't used anywhere else.  We should discuss this prior to making more changes on this issue.

### CODE REVIEW

**Review by jimfenton — COMMENTED** [review]
"issuing department or agency" seems like a mouthful. Is there any instance where "issuing agency" isn't sufficient?

I thought we had avoided this issue by our current wording, "organization that manages the PIV account", but apparently not. Alternative idea: go back to using "issuing agency" and then provide a definition in the glossary that makes it clear who that is.

**Comment by regenscheid** [comment]
Still need to revise the intro so that it also uses the term "issuing department or agency" when describing the concepts of PIV accounts and derived credentials.

**Comment by regenscheid** [comment]
I didn't make very many additional changes, but I would now be comfortable merging this. As described on a earlier call, the intent is to emphasize that the the PIV account, and thus the PIV cards, DPCs, and IDMS records, is under the control of the issuing **department or agency.**   This is intended to reduce confusion when an agency relies on a third party (e.g., a shared service provider like USAccess) for card issuance. In such cases, the agency is still the issuer, not USAccess.

File: _FIPS201/requirements.md [hunk 1]
```diff
@@ -601,7 +601,9 @@ Derived PIV credentials are additional PIV credentials that are issued based on
...
+[[SP 800-157]](../_Appendix/references.md#ref-SP-800-157). Derived PIV credentials **SHALL** meet the requirements for Authenticator Assurance Level (AAL) 2 or 3 specified in [[SP 800-63B]](../_Appendix/references.md#ref-SP-800-63B). All derived PIV credentials meeting AAL2 but not AAL3 requirements **SHALL** be supported at AAL2 only. Note that derived PIV credentials meeting AAL3 requirements also fulfill the requirements of AAL2 and can be used in circumstances requiring AAL2. The issuer **SHALL** attempt to promptly notify the cardholder of the binding of a derived PIV credential through an independent means that would not afford an attacker an opportunity to erase the notification. More than one independent notification method **MAY** be used to ensure prompt receipt by the cardholder.
```
Comment (jimfenton) [inline _FIPS201/requirements.md:604]: "The issuer **SHALL** attempt..." Double checking to make sure that instance wasn't intended to also be "issuing department or agency". I think it's fine as is; doesn't really matter if it's the department or agency, or a service provider sending the notice.

**Review by jimfenton — APPROVED** [review]
Small question on one phrase

**Review by hferraio — APPROVED** [review]

File: _FIPS201/requirements.md (hunk 1 above)
Comment (regenscheid) [reply _FIPS201/requirements.md:604]: Correct- it doesn't matter. No change.

**Pull request merged**